File: app/src/main/java/ch/bailu/aat/map/layer/control/CustomBarLayer.java
package ch.bailu.aat.map.layer.control;

import ch.bailu.aat.map.MapColor;
import ch.bailu.aat.map.MapContext;
import ch.bailu.aat.views.bar.ControlBar;


public class CustomBarLayer extends ControlBarLayer {



    public CustomBarLayer(MapContext mc, ControlBar b) {
        super(mc,b, TOP, MapColor.DARK);
        showBar();
    }

    @Override
    public void drawInside(MapContext mcontext) {

    }

    @Override
    public void onAttached() {

    }

    @Override
    public void onDetached() {

    }
}

File: aat-android/src/main/kotlin/ch/bailu/aat/map/layer/CustomBarLayer.kt
package ch.bailu.aat.map.layer

import ch.bailu.aat.util.ui.theme.UiTheme
import ch.bailu.aat.views.bar.ControlBar
import ch.bailu.aat_lib.map.MapColor
import ch.bailu.aat_lib.map.MapContext
import ch.bailu.aat_lib.map.edge.Position

class CustomBarLayer(mc: MapContext, b: ControlBar, theme: UiTheme) : ControlBarLayer(
    mc,
    b,
    Position.TOP,
    MapColor.setAlpha(theme.getBackgroundColor(), MapColor.ALPHA_LOW)
) {
    init {
        showBar()
    }

    override fun drawInside(mcontext: MapContext) {}
    override fun onAttached() {}
    override fun onDetached() {}
}
